docs/src/developer/release.rst

```diff
@@ -0,0 +1,62 @@
+*********
+Releasing
+*********
+
+In this document, we describe the procedure used to release a new version of Oríon to the public.
+Release artifacts are distributed through PyPI_.
+
+Creating a release candidate
+============================
+The first step in releasing a new version is to create a release candidate. A release candidate
+allows us to throughly test the new version and iron out the remaining bugs. Additionally, it's also
+at this time that we make sure to change the version number and update related documentation such as
+the README.md.
+
+#. Create a new branch from the *develop* branch named ``release-{version}rc``, where
+   ``{version}`` is replaced by the number of the new version (e.g., ``1.2.0``). This effectively
+   freezes the feature set for this new version, while allowing regular development to continue take
+   place in the *develop* branch. More information is available in :ref:`standard-vcs`.``{version}
```

Comment:
```suggestion
   place in the *develop* branch. More information is available in :ref:`standard-vcs`.``{version}``
```

Reply:
Well, why is ``{version}`` there? Is it a mistake?

Reply:
Mmmmmh, I'm not sure how this guy arrived here. Good catch.